ansible/playbooks/AdoptOpenJDK_Unix_Playbook/roles/OpenSSL/tasks/main.yml

```diff
@@ -0,0 +1,109 @@
+---
+###################
+# OpenSSL v1.1.0g #
+###################
+# Required by OpenJ9 for out-of-process JIT compilation aka JITasS
```

Comment:
Minor point but should this have been `JITaaS` rather than `JITasS`?

Reply:
right

Reply:
fixed